api/services/gameService.js

```diff
@@ -247,6 +247,8 @@ module.exports = {
                 Card.destroy({
                   or: deleteCardsCriteria,
                 }),
+                // Inform all clients this game is over
+                sails.sockets.blast('gameFinished', { gameId: game.id }),
```

Comment:
This is great. Much cleaner than my previous implementation which cluttered all the individual endpoints with the need for this blast